setup.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: page
+title: Set up GUAC with Docker Compose
+permalink: /setup/
+nav_order: 2
+---
+
+
+# Set up GUAC with Docker Compose
+
+{: .note }
+If you’d prefer, you can set up [GUAC with Kubernetes](https://github.com/kusaridev/helm-charts/tree/main/charts/guac) instead.
+
+
+GUAC consists of multiple components. You may have seen a subset of these being used into various [GUAC demos](demo page link). To get the most value out of GUAC, you’ll need to set up all components. This tutorial will walk you through how to deploy GUAC, using Docker Compose, so that you get the full set of components.
+
+If you’re curious about the various GUAC components and what they do, see [How GUAC components work together](link to new article under GUAC details).
+
+{: .no_toc }
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+- TOC
+{:toc}
+</details>
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [Git](https://github.com/git-guides/install-git)
+- [Go](https://go.dev/doc/install) 
+- [Make](https://www.gnu.org/software/make/)
+
+## Step 1: Clone GUAC
+1. Clone GUAC to a local directory:
+  ```bash
+  git clone https://github.com/guacsec/guac.git
+  ```
+
+2. Optional: If you want test data to use, clone GUAC’s test data:
+  ```bash
+  git clone https://github.com/guacsec/guac-data.git
+  ```
+
+3. Go to your GUAC directory (the rest of the steps need to be done from this directory):
+  ```bash
+  cd guac
+  ```
+
+## Step 2: Build the containers
+
+From your GUAC directory, run:
+
+  ```bash
+  make container
+  ```
+
+## Step 3: Start the GUAC server
+
+  1. In another terminal, from your GUAC directory, run:
+
+  ```bash
+  docker-compose up
+  ```
+  2. Verify that GUAC is running:
+  ```bash
+  docker compose ls --filter "name=guac"
+  ```
+
+  You should see:
+
+  ```bash
+  NAME                STATUS              CONFIG FILES
+  guac                running(7)          /Users/lumb/go/src/github.com/guacsec/guac/docker-compose.yml
+  ```
+
+  **If you don’t see the above,** run `docker-compose down` and try starting up GUAC again. Because Docker Compose caches the containers used, the unclean state can cause issues.
+
+### GUAC Ports
+
+|---
+| Port Number | GUAC Component | Note
+|-|:-|:-:|-:
+| 8080 | GraphQL server | To see the GraphQL playground, <br />visit [http://localhost:8080](http://localhost:8080)  
+|---
```

Comment:
This line isn't showing in the table. Maybe a space is needed before it?

Reply:
I see it. Not sure if Hayley fixed it.